network/peer/tls_config.go

```diff
@@ -24,7 +23,7 @@ func TLSConfig(cert tls.Certificate, keyLogWriter io.Writer) *tls.Config {
 		//
 		// During our security audit by Quantstamp, this was investigated
 		// and confirmed to be safe and correct.
-		InsecureSkipVerify: true,
+		InsecureSkipVerify: true, // #nosec G402
```

Comment:
I know we weren't doing this correctly before... But I think it's standard to have comments for automated systems not to start with a space.
```suggestion
		InsecureSkipVerify: true, //#nosec G402
```

Reply:
I wasn't aware of that standard, but don't have any reason not to adhere to that.

Reply:
I think I first saw this here: https://golangci-lint.run/usage/false-positives/

> Use `//nolint` instead of `// nolint` because machine-readable comments should have no space by Go convention.

There is probably a reference in a more "standard" golang place... but I couldn't quickly find it.